less/plugins/theme-color-level.js

```diff
@@ -54,9 +48,9 @@ functions.add('theme-color-level', function ({ value: colorName }, { value: leve
 
 	// If `themeColors` hasn’t been defined yet, set it to the value of `@theme-colors`.
 	if (Object.keys(themeColors).length === 0)
-		themeColors = listToMap(lookupVariable(context, '@theme-colors'))
+		themeColors = rulesetToMap(lookupVariable(context, '@theme-colors'))
 
-	const color      = themeColors[colorName]
+	const color      = themeColors[colorName].type === 'Expression' ? themeColors[colorName].eval(context) : themeColors[colorName] // .eval() gets the Color node that the Expression node refers to
```

Comment:
What do you think about something like this, just to make it a bit more readable?
```suggestion
	let color = themeColors[colorName]
	
	if (color.type === 'Expression')
		color =  color.eval(context) // .eval() gets the Color node that the Expression node refers to
	
```